lib/config-helper.js

```diff
@@ -0,0 +1,360 @@
+
+var fs = require('fs-extra');
+var pathExtra = require('path-extra');
+var configWatcher = require('./config-watcher');
+
+module.exports = function(){
+
+  var config;
+  var home = '';
+  var mainAppHost = 'localhost';
+  var mainAppPort = 19104;
+  var appsPort = 18001;
+  //var loadedPlugins = {};
```

Comment:
if you don t see problems on your side with this change on your side,  i ll proceed. Then i ll have to update the dashboard i made weeks ago now.
The change consist of moving export* methods to respective plgin/application helpers.